browser/brave_search/brave_search_browsertest.cc

```diff
@@ -41,7 +41,7 @@ namespace {
 
 const char kEmbeddedTestServerDirectory[] = "brave-search";
 const char kAllowedDomain[] = "search.brave.com";
-const char kAllowedDomainDev[] = "search-dev.brave.com";
+const char kAllowedDomainDev[] = "search-dev-local.brave.com";
```

Comment:
Not sure if we want this to be exhaustive but the main "dev" domain is `search.brave.software` and `search-dev-local.brave.com` is only used for local development.

Reply:
thanks, I will change it to `search.brave.software`